gstreamer-player/src/auto/player.rs

```diff
@@ -282,8 +282,7 @@ impl Player {
 
     pub fn get_property_suburi(&self) -> Option<String> {
         unsafe {
-            let mut value = Value::uninitialized();
-            gobject_ffi::g_value_init(value.to_glib_none_mut().0, <String as StaticType>::static_type().to_glib());
+            let mut value = Value::from_type(<String as StaticType>::static_type());
```

Comment:
This requires GIT master for glib-rs, I'd prefer not to depend on that as I want to make a 0.10 release soonish without waiting for glib-rs to be released first. Can you use the version of gir that was used last time, d50d839.